config/config/wc-config.yaml

```diff
@@ -236,7 +236,19 @@ networkPolicies:
     wcNodes:
       ips: # ip of all nodes in the cluster for internal communication
         - set-me
+    externalLoadBalancer: false
+  monitoring:
+    enabled: true
+
   certManager:
     enabled: true
-  monitoring:
+  # letsencrypt ip addresses
+    letsencrypt:
+      ips:
+        - set-me
+
+  ingressNginx:
     enabled: true
+    allowList:
+      ips:
```

Comment:
Same here, move to common-config if it's present on both

Reply:
but this is different, in case of sc and wc 

in case of sc , it would be our customer ips that they use to access our services 

in case of wc, it would be their customers ips to use their(our customer's) services. 

I mean, we could have it in common config, but it would be nice to differentiate. do you think so ?

Reply:
Right, yes that's true. So in the case where we want the same list for both we could move it to common, but by default it's in separate files since they are most probably not the same.

But at least the `certManager` section could be moved. That should always be the same between clusters